src/modules/sub-items/components/bulk-move/bulk.move.btn.js

```diff
@@ -0,0 +1,138 @@
+
+        removeItemsFromList(movedLocationsIds);
+
+        if (notMoved.length) {
+            const message = Translator.trans(
+                /*@Desc("You do not have permission to move at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.")*/ 'bulk_move.error.message',
+                {},
+                'sub_items'
+            );
+
+            window.eZ.helpers.notification.showErrorNotification(message);
+        }
+
+        if (movedLocations.length) {
+            const message = Translator.trans(
+                /*@Desc("The selected content item(s) have been sent to <u>%location_name%</u>")*/ 'bulk_move.success.message',
+                { location_name: location.ContentInfo.Content.Name },
+                'sub_items'
+            );
+
+            window.eZ.helpers.notification.showSuccessNotification(message);
+        }
+    }
+
+    toggleUdw(show) {
+        this.setState(() => ({
+            isUdwOpened: show,
+        }));
+    }
+
+    closeUdw() {
+        this.toggleUdw(false);
+    }
+
+    onUdwConfirm([selectedLocation]) {
+        this.closeUdw();
+        this.bulkMove(selectedLocation);
+    }
+
+    renderUdw() {
+        const { isUdwOpened } = this.state;
+
+        if (!isUdwOpened) {
+            return null;
+        }
+
+        const UniversalDiscovery = window.eZ.modules.UniversalDiscovery;
+        const { restInfo, parentLocationId, selectedItems, udwConfig } = this.props;
+        const selectedItemsLocationsIds = selectedItems.map((item) => item.location.id);
+        const excludedMoveLocations = [parentLocationId, ...selectedItemsLocationsIds];
+        const title = Translator.trans(/*@Desc("Choose location")*/ 'udw.choose_location.title', {}, 'sub_items');
+        const udwProps = {
+            ...udwConfig,
+            title,
+            restInfo,
+            onCancel: this.closeUdw,
+            onConfirm: this.onUdwConfirm,
+            canSelectContent: ({ item }, callback) => {
+                callback(!excludedMoveLocations.includes(item.id));
+            },
+            allowContainersOnly: true,
+        };
+
+        return ReactDOM.createPortal(<UniversalDiscovery {...udwProps} />, this.udwContainer);
+    }
+
+    render() {
+        const { selectedItems } = this.props;
+        const label = Translator.trans(/*@Desc("Move selected items")*/ 'move_btn.label', {}, 'sub_items');
+        const disabled = !selectedItems.length;
+        const baseClassName = 'm-sub-items__btn';
+        let className = `${baseClassName} ${baseClassName}--move`;
+
+        className = disabled ? `${className} ${baseClassName}--disabled` : className;
+
+        return (
+            <Fragment>
+                <div className={className} title={label} onClick={this.onBtnClick}>
+                    <svg className="ez-icon ez-icon--medium">
+                        <Icon name="move" extraClasses="ez-icon--medium" />
```

Comment:
`Icon` inside `svg`? this will be `svg` inside `svg`.

Reply:
You're right! I was too fast. :)